VAMobile/.eslintrc.js

```diff
@@ -51,5 +51,11 @@ module.exports = {
         memberSyntaxSortOrder: ['none', 'all', 'multiple', 'single'],
       },
     ],
+    'react/no-unstable-nested-components': [
+      'warn',
+      {
+        allowAsProps: true,
+      }
+    ]
```

Comment:
This rule was automatically enabled from the ESLint updates, probably from one of the plugin's configs. It caused about a dozen failures, which most of them I was able to fix manually, but I had to enable [allowAsProps](https://github.com/jsx-eslint/eslint-plugin-react/blob/master/docs/rules/no-unstable-nested-components.md#rule-options) so that we can pass components into react navigation props like we are [here](https://github.com/department-of-veterans-affairs/va-mobile-app/blob/776e30ba3f1f59136f91eb2ce79493a50963b941/VAMobile/src/components/Carousel/Carousel.tsx#L64-L66) and [here](https://github.com/department-of-veterans-affairs/va-mobile-app/blob/4368c5b33b6faf77ab1a652aa21692d59baac0ef/VAMobile/src/screens/WebviewScreen/WebviewScreen.tsx#L97-L99)